src/Fist/Database/tests/DatabaseConnectionTest.php

```diff
@@ -188,7 +194,9 @@ public function testDriverSwapping()
     public function testSharedDriverConnectionsUpdateSettings()
     {
         $db = new Database(
-            $repository = new ContainerRepository($container = new Container(), 'db.')
+            $repository = new ContainerRepository(
+                new Container(), 'db.'
```

Comment:
__[code_sniffer]__
* Only one argument is allowed per line in a multi-line function call

<!-- Comment by [SideCI](https://sideci.com) -->